model-engine/tests/unit/domain/test_llm_use_cases.py

```diff
@@ -953,3 +958,53 @@ async def test_delete_public_inference_model_raises_not_authorized(
         await use_case.execute(
             user=user, model_endpoint_name=llm_model_endpoint_sync[0].record.name
         )
+
+
+@pytest.mark.parametrize(
+    "framework",
+    [
+        LLMInferenceFramework.DEEPSPEED,
+        LLMInferenceFramework.LIGHTLLM,
+        LLMInferenceFramework.TEXT_GENERATION_INFERENCE,
+        LLMInferenceFramework.VLLM,
+    ],
+)
+@pytest.mark.parametrize("image_tag", ["0.9.3-launch_s3"])
+@pytest.mark.parametrize(
+    "checkpoint_path",
+    [
+        "s3://scale-ml/testing/llama-2-13b-bin-path/",
+        "s3://scale-ml/testing/llama-2-13b-safetensors-path/",
+    ],
+)
+def test_load_model_weights_sub_commands(
```

Comment:
I should've been more specific about the unit tests. It looks like this unit test calls the live use case and endpoint service instead of using the fake endpoint service pytest fixture. I think that this is fine, but we might want to treat it more like an integration test that runs against our live infrastructure instead of a unit test (since it implicitly relies on other components working for this test to pass).